concepticondata/retired.json

```diff
@@ -29949,7 +29949,2057 @@
-    }
+    },
+	{
+	  "id": "Seifart-2015-410-1",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-1"
+	},
+	{
+	  "id": "Seifart-2015-410-2",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-2"
+	},
+	{
+	  "id": "Seifart-2015-410-3",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-3"
+	},
+	{
+	  "id": "Seifart-2015-410-4",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-4"
+	},
+	{
+	  "id": "Seifart-2015-410-5",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-5"
+	},
+	{
+	  "id": "Seifart-2015-410-6",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-6"
+	},
+	{
+	  "id": "Seifart-2015-410-7",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-7"
+	},
+	{
+	  "id": "Seifart-2015-410-8",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-8"
+	},
+	{
+	  "id": "Seifart-2015-410-9",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-9"
+	},
+	{
+	  "id": "Seifart-2015-410-10",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-10"
+	},
+	{
+	  "id": "Seifart-2015-410-11",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-11"
+	},
+	{
+	  "id": "Seifart-2015-410-12",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-12"
+	},
+	{
+	  "id": "Seifart-2015-410-13",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-13"
+	},
+	{
+	  "id": "Seifart-2015-410-14",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-14"
+	},
+	{
+	  "id": "Seifart-2015-410-15",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-15"
+	},
+	{
+	  "id": "Seifart-2015-410-16",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-16"
+	},
+	{
+	  "id": "Seifart-2015-410-17",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-17"
+	},
+	{
+	  "id": "Seifart-2015-410-18",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-18"
+	},
+	{
+	  "id": "Seifart-2015-410-19",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-19"
+	},
+	{
+	  "id": "Seifart-2015-410-20",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-20"
+	},
+	{
+	  "id": "Seifart-2015-410-21",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-21"
+	},
+	{
+	  "id": "Seifart-2015-410-22",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-22"
+	},
+	{
+	  "id": "Seifart-2015-410-23",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-23"
+	},
+	{
+	  "id": "Seifart-2015-410-24",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-24"
+	},
+	{
+	  "id": "Seifart-2015-410-25",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-25"
+	},
+	{
+	  "id": "Seifart-2015-410-26",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-26"
+	},
+	{
+	  "id": "Seifart-2015-410-27",
+	  "comment": "renaming",
+	  "replacement": "Seifart-2015-408-26"
```

Comment:
The remapping is unfortunately wrong here (did you generate IDs before removing the doubles?): this maps the old ID for `HEART` to the new ID for `BLOOD` and propagates the wrong ID throughout the following entries.